data/wanderers middle.txt

```diff
@@ -3035,59 +3035,53 @@ event "wanderers: exiles hostile"
 
 event "wanderers: exiles have drones"
 	fleet "Korath Home"
-		variant 12
-			"Korath World-Ship"
-		variant 12
-			"Korath World-Ship B"
-		variant 12
-			"Korath World-Ship C"
-		variant 2
+		add variant 2
```

Comment:
To make this work properly, the original "Korath Home" definition (in korath.txt) will need to be updated so that each world-ship variant has a weight of 12 instead of 1.